basyx.submodelrepository/basyx.submodelrepository-core/src/test/java/org/eclipse/digitaltwin/basyx/submodelrepository/core/SubmodelRepositorySuite.java

```diff
@@ -523,7 +523,7 @@ private SubmodelElement getExpectedSubmodelElement() {
 				.get();
 	}
 
-	private Submodel buildDummySubmodel(String id) {
+	protected Submodel buildDummySubmodel(String id) {
```

Comment:
When the test cases are moved in this class (as suggested above) then this change is not required.

Reply:
done